business_requirement_deliverable/tests/test_br.py

```diff
@@ -149,7 +149,13 @@ def test_open_business_requirement_dl_rl(self):
         self.br.open_deliverable_line()
         self.br.open_resource_line()
 
+    def test_compute_total_revenue_cost(self):
+        self.br._compute_dl_total_revenue()
+        self.br._compute_rl_total_cost()
```

Comment:
You're not actually testing the code here.
You should add lines to a BR, and then read the totals and compare with the expected value...